app/controllers/facility_reservations_controller.rb

```diff
@@ -121,8 +121,11 @@ def new
 
   # POST /facilities/:facility_id/instruments/:instrument_id/reservations
   def create
-    @instrument   = current_facility.instruments.find_by_url_name!(params[:instrument_id])
-    @reservation  = @instrument.reservations.new(params[:reservation])
+    @instrument =
+      current_facility.instruments.find_by!(url_name: params[:instrument_id])
+    @reservation = @instrument.admin_reservations.new
+    @reservation.assign_attributes(params[:reservation])
```

Comment:
That was my first thought, but I went with the pattern we're using in `ReservationsController#create` (half expecting CodeClimate to complain about duplication).